source/blog/2016-08-03-fedora-review-tool-for-reviewing-rdo-packages.html.md

```diff
@@ -0,0 +1,78 @@
+---
+title: fedora-review tool for reviewing RDO packages
+author: chandankumar
+tags: dlrn,rdo,openstack,packaging,centos,fedora,fedora-review
+date: 2016-08-03 16:05:19 IST
+---
+
+This tool makes reviews of rpm packages for Fedora easier. It tries to automate most of the process.
+Through a bash API the checks can be extended in any programming language and for any programming language.
+
+We can also use it for also reviewing RDO packages on Centos 7/Fedora-24.
+
+## Install fedora-review and DLRN
+
+[1.] Install fedora-review
+
+**For Centos 7**
+
+Enable epel repos on centos
+
+	$ sudo yum -y install epel-release
+
+Download fedora-review el7 build from Fedora Koji
+
+	$ sudo yum -y install https://kojipkgs.fedoraproject.org//packages/fedora-review/0.5.3/2.el7/noarch/fedora-review-0.5.3-2.el7.noarch.rpm
+
+**On Fedora 24**
+
+	$ sudo dnf -y install fedora-review
+
+[2.] Add the user you intend to run as to the mock group:
+
+	$ sudo usermod -a -G mock $USER
```

Comment:
mock needs to be installed, so we can add the user to the group. Maybe we should install it at the sme time as fedora-review?